config/routes.rb

```diff
@@ -3,9 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-  resources :merchants, param: :id do
+  resources :merchants, param: :merchant_id do
```

Comment:
When I had this resources using param: :merchant_id, it created some odd looking paths with merchant_merchant_id, are you getting some like this too?

Reply:
I got the same result on your branch but not on mine. We may need to jump on a pair sesh to troubleshoot because I couldn't solve your branch with my syntax. The only thing different about yours than mine is the `root` and `admin` routes in yours...

Reply:
We will have to solve the conflict closer to primary branch merge